### PR TITLE
Fixed instructions for the field

### PR DESCRIPTION
**Pull request by spficklin**
Issue #625 and discovered bug.

## Type(s) of Change(s)

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] API-specific change (fix or addition to an API function)
- [ ] Updates documentation (inline or markdown files)

## Description

This addresses issue #625 where the instructions for setting field values for the remote__data field were not clear enough.  The problem was that the `Available Tokens` fieldset appeared below the field named `Field to Display` and it implied that tokens could be used.  This is not the case.  To fix this I

1) Moved the `Available Tokens` fieldset below the `Query to Execute` field.
2) The instructions needed more details and were becoming too long so I added fieldsets to contain the instructions for both fields.  

Also, there was a 1-line bug fix in web services that I included in this PR. You can see it in the code review. It was so minor I decided to include it here rather than create it's on PR.  It fixes the problem when browsing web services.  For fields that are not auto attached the web services gives a link.  If the user follows that link it takes them to another web service result that then provides the values.  This result was not complete.  This fixes it.

## Testing?

There is no need to test the fixes to the remote__data field unless you want to see the changes yourself. Otherwise a code review and looking at the screenshots should suffice.  

To test the web services fix, pull the branch, navigate through web services to any entity that has relationships and follow the relationships link. You should see all of the relationships.

## Screenshots (if appropriate):
Screenshot of new remote__data form page with described fixes:

image

Screenshot of web services for a gene with non-attached fields shown as links:

image

Screenshot of web services when looking at relationships, before the fix.

image

And after the fix:

image

## Additional Notes (if any):

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by laceysanderson — APPROVED** [review]
I changed the wording a little bit on "Query to Execute" instructions:
> If the full URL to the remote tripal content web service is "https://[tripal_site]/web-services/content/v0.1/". Then this field should contain the text immediately after the "content/v0.1" portion of the URL. For information about building web services queries see the online documentation at The Tripal v3 User's Guide. For example, suppose this field is attached to an Organism content type on the local site, and you want to retrieve a field for the same organism on a remote Tripal site. To retrieve the matching record, you will want to query on the genus and species, since it is unique and, you want them to match the organism for each specific local organism page. You can use tokens to do this (see the "Available Tokens" fieldset below). For this example, the full remote web service endpoint would be "https://[tripal_site]/web-services/content/v0.1/Organism" and the query text should be "Organism?genus=[taxrank__genus]&species=[taxrank__species]".
screen shot 2018-09-28 at 8 55 58 pm

And switched the example on the "Field to Display" to common name so that it was visible in the JSON without scrolling.
screen shot 2018-09-28 at 8 57 34 pm

With those changes, I approve this PR as fixing my original concerns :-) Thanks @spficklin!

**Comment by laceysanderson** [comment]
I think this change is small enough (not many lines of code) that it only requires one review. As I've tested and approved it, it can be merged once @spficklin approves my small changes :-)

**Comment by spficklin** [comment]
Thanks @laceysanderson

**Pull request merged**
